core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/version/JavaVersion.java

```diff
@@ -150,6 +150,25 @@ public boolean isGreaterOrEqualThan(JavaVersion otherVersion) {
 		return !this.isEarlyAccess || otherVersion.isEarlyAccess;
 	}
 
+	/**
+	 * Compare another version instance with this instance disregarding early access status.
+	 *
+	 * @param otherVersion
+	 *            version to compare with
+	 * @return {@code true} if this instance is greater than or equal to {@code otherVersion}
+	 */
+	public boolean isGreaterOrEqualThanDisregardEa(JavaVersion otherVersion) {
+		int maxLength = Math.max(versionNumbers.length, otherVersion.versionNumbers.length);
+		for (int i = 0; i < maxLength; i++) {
+			int thisNumber = versionNumbers.length > i ? versionNumbers[i] : 0;
+			int otherNumber = otherVersion.versionNumbers.length > i ? otherVersion.versionNumbers[i] : 0;
+			if (thisNumber != otherNumber) {
+				return thisNumber > otherNumber;
+			}
+		}
```

Comment:
It would be nice to avoid code duplication with the above method `isGreaterOrEqualThan`

Reply:
Also I think the copyright year need to be bumped to 2021 for this file

Reply:
I addressed these in the fixup commits. Let me know if you agree with the changes from the fixups and then I will squash them and force push again.